test/libsolidity/InlineAssembly.cpp

```diff
@@ -83,6 +109,21 @@ void parsePrintCompare(string const& _source)
 
 }
 
+#define CHECK_ERROR(text, assemble, typ, substring) \
```

Comment:
I actually changed that to `type` only to notice that it will result in `Error::type` to be replaced by the macro argument, so I think this was done intentionally by @pirapira :-)